pep-0680.rst

```diff
@@ -289,8 +289,9 @@ be worked around in user code, or by using a third-party library.
 
 The proposed API takes a binary file, while ``toml.load`` takes a text file and
 ``json.load`` takes either. Using a binary file allows us to ensure UTF-8 is
-the encoding used, and avoid incorrectly parsing single carriage returns as
-valid TOML due to universal newlines in text mode.
+the encoding used (ensuring correct parsing on platforms with other default
+encodings, such as Windows), and avoid incorrectly parsing single carriage
+returns as valid TOML due to universal newlines in text mode.
```

Comment:
```suggestion
encodings, such as Windows), and avoid incorrectly parsing files using a
single carriage return (``\r``) for the end of line character as valid TOML
(due to Python's universal newline conversion in text mode).
```

Despite careful re-reading, I was initially confused by this line when reading the original PEP, and then misunderstood it to mean files that were just single `CR`s, rather than files _that use single `CR`s as the newline character_. I suggest clarifying this as above.

Reply:
Added a commit to clarify this. I think this reason is much less important than the other one, so don't wish to spend too many words on this.

Reply:
The commit doesn't really clarify the confusion I had over its intended meaning, as the problematic usage is with using a single `CR` _as an end of line character_, not single CRs anywhere in the file being an invalid character as the current wording implies (which both won't trigger the problem unless it is the EOL character, and is legal TOML inside multi-line double-quoted strings, at least per the [verbatim letter of the spec](https://toml.io/en/v1.0.0#string) if I'm reading it correctly).

If it isn't that important, such that it wouldn't be worth adding a few extra words to make its intended meaning clear and explicit and avoid introducing additional confusion, then maybe it would be best to just remove it entirely?

Reply:
I don't have a preference on the wording here, but I do think this is important and should not be removed.

Regarding TOML spec confusion: note that CR character is _always_ part of an EOL sequence (CR+LF). If it isn't then the TOML is invalid. A bare CR is not valid in multi-line strings (double or single quoted) as per [toml.abnf](https://github.com/toml-lang/toml/blob/master/toml.abnf).

Reply:
> I don't have a preference on the wording here, but I do think this is important and should not be removed.

In that case, I suggest including the above clarification (or similar), as the present wording was the source of confusion on what you meant here (by someone who makes heavy use of TOML, though lacking your depth of expertise into the intricacies of the format).

> Regarding TOML spec confusion: note that CR character is always part of an EOL sequence (CR+LF). If it isn't then the TOML is invalid. A bare CR is not valid in multi-line strings (double or single quoted) as per toml.abnf.

Ah, thanks for the clarification. I was mislead by this line,

> Any Unicode character may be used except those that must be escaped: backslash and the control characters other than tab, line feed, and carriage return (U+0000 to U+0008, U+000B, U+000C, U+000E to U+001F, U+007F).

and the lack of explicitly requiring any `CF` to always be followed by `LF`, but indeed the formal ABNF is correct.